pages/index.tsx

```diff
@@ -5,6 +5,7 @@ import GetAllProjectsMeta from '../src/utils/getMetaTags/GetAllProjectsMeta';
 import getStoredCurrency from '../src/utils/countryCurrency/getStoredCurrency';
 import { getRequest } from '../src/utils/apiRequests/api';
 import DirectGift from '../src/features/donations/components/treeDonation/DirectGift';
+import currencyContext from '../src/utils/Context/CurrencyContext';
```

Comment:
Isn't this creating a different context as the one created in _app.tsx?

Reply:
No, context is created in the context file `src/utils/Context/CurrencyContext.js`, a provider is created in `_app.tsx` so that the context is accessible anywhere in the app.

Reply:
then you can use `useContext` to get the value and the setter function.

Reply:
Ok, it's a singleton pattern. Sorry, sometimes I am still getting confused about this.